tests/e2e/transitions/sentinel_api_transitions.spec.js

```diff
@@ -486,11 +486,13 @@ describe('transitions', () => {
         chai.expect(doc.contact._id).to.equal('chw1');
 
         chai.expect(messages.messages.length).to.equal(9);
-        chai.expect(messages.messages.length).to.equal(docs.reduce((sum, doc) => sum + doc.tasks.length, 0));
+        chai.expect(docs.reduce((sum, doc) => sum + doc.tasks.length, 0)).to.equal(10);
         docs.forEach(doc => {
           doc.tasks.forEach(task => {
             task.messages.forEach(message => {
-              chai.expect(messages.messages.find(m => m.id === message.uuid)).to.be.ok;
+              if (message.message) {
```

Comment:
Do we have messages that don't have a `message` property? Is it because the translation key is not translated? 
Does that mean that every project that will upgrade to this version will require a new translation key for the new messages that `update_clinics` sends? 
We should add this new system-side message to our translations. 
Also, the exclusion of the new message that `update_clinics` is generated should be more nuanced - the reader should understand which message we are expecting and not comparing to the configured ones.